core/new-gui/src/app/workspace/types/workflow-websocket.interface.ts

```diff
@@ -68,7 +72,8 @@ export type TexeraWebsocketEventTypeMap = {
   'RecoveryStartedEvent': {},
   'BreakpointTriggeredEvent': BreakpointTriggerInfo,
   'ModifyLogicCompletedEvent': {},
-  'OperatorCurrentTuplesUpdateEvent': OperatorCurrentTuples
+  'OperatorCurrentTuplesUpdateEvent': OperatorCurrentTuples,
+  'ErrorEvent': BackendError
```

Comment:
How about `WorkflowExecutionErrorEvent` to rename `ErrorEvent` and `WorkflowExecutionError` to rename `BackendError`

Reply:
Agreed